com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderIncludeCompiler.java

```diff
@@ -0,0 +1,207 @@
+package com.dynamo.bob.pipeline;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.regex.Matcher;
+import java.io.IOException;
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.dynamo.bob.CompileExceptionError;
+import com.dynamo.bob.Project;
+import com.dynamo.bob.fs.IResource;
+import com.dynamo.bob.pipeline.ShaderUtil.Common;
+
+/* ShaderIncludeCompiler
+ * ========================
+ * This class contains functionality to support #include "path-to-file" directives from shader files.
+ * Includes must be in double quotes, <> and '' are currently not supported, and the included file
```

Comment:
Using the regex mentioned below, we should be able to support <>
The '  is for characters in C and I don't think that's something we want to dabble with.